src/dfx/src/actors/btc_adapter.rs

```diff
@@ -90,7 +90,10 @@ impl BtcAdapter {
 
         let (sender, receiver) = unbounded();
 
-        let handle = btc_adapter_start_thread(logger, self.config.clone(), addr, receiver)?;
+        let handle = anyhow::Context::context(
```

Comment:
I'm sure there's a good reason, but why does this the form `anyhow::Context::context(do_something(), "context")?`; instead of the form `do_something().context("context")?` used elsewhere in this PR?

Reply:
Namespace clashes. This file imports `actix::Context`, so I can't also import `anyhow::Context`. If there's a better solution/Rust feature to solve this I'd be happy to learn about it